libkodimote/tvshows.h

```diff
@@ -43,6 +43,7 @@ class TvShows : public KodiLibrary
 
     ThumbnailFormat thumbnailFormat() const { return ThumbnailFormatLandscape; }
     bool allowWatchedFilter() { return true; }
+    QString watchedFilterSetting() { return "showWatchedTvShows"; }
```

Comment:
If I understand correctly, "no". As TVShows, Seasons and Episodes use the same setting. You don't want to use different settings for these, and Kodi also uses the same. When you set `Hide watched` in the TV Show listing and you open a show it also automatically hides the watched seasons (and same for episodes), which also works the other way around, changing the setting in episodes and going up to seasons means the watched seasons are also hidden immediately.